openedx/core/djangoapps/programs/utils.py

```diff
@@ -306,6 +315,11 @@ def _attach_course_run_is_enrollment_open(self, run_mode):
         enrollment_end = self.course_overview.enrollment_end or datetime.datetime.max.replace(tzinfo=utc)
         run_mode['is_enrollment_open'] = self.enrollment_start <= datetime.datetime.now(utc) < enrollment_end
 
+    def _attach_course_run_course_price(self, run_mode):
+        self.data['number_of_courses'] += 1
+        self.data['full_program_price'] += float(run_mode['seats'][0]['price'])
```

Comment:
The program detail page is showing a 500 error on the [sandbox](https://hms.sandbox.edx.org/programs/7f5526fc-07bd-45af-b65c-2f3a6d6e098c/detail) because one of the course runs configured in the program does not have seat information. We should handle this condition here.